Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -85,26 +105,53 @@ public struct OfflineMapAreasView: View {
                 emptyPreplannedMapAreasView
             }
         case .failure(let error):
-            VStack(alignment: .center) {
-                Image(systemName: "exclamationmark.circle")
-                    .imageScale(.large)
-                    .foregroundStyle(.red)
-                Text(error.localizedDescription)
-            }
-            .frame(maxWidth: .infinity)
+            errorView(error)
         case .none:
             ProgressView()
                 .frame(maxWidth: .infinity)
         }
     }
     
-    @ViewBuilder private var emptyPreplannedMapAreasView: some View {
+    @ViewBuilder private var offlinePreplannedMapAreasView: some View {
+        if let models = mapViewModel.offlinePreplannedMapModels {
+            if !models.isEmpty {
+                List(models) { preplannedMapModel in
+                    PreplannedListItemView(model: preplannedMapModel, selectedMap: $selectedMap, onDeletion: {
+                        Task { await makePreplannedMapModels() }
+                    })
```

Comment:
I'm wondering if the on deletion closure isn't necessary. Is there a way that a task modifier could be used here? Something like this:

```swift
PreplannedListItemView(model: preplannedMapModel, selectedMap: $selectedMap)
    .task(preplannedMapModel.status) { // Does not work since status is not equatable.
        guard preplannedMapModel.status == .needsNewModels else { return }
        await makePreplannedMapModels()
    }
```

But then having thought of it some more, this really does seem like something the model itself should do, right? I just think there needs to be a way to coalesce those updates. Think of the scenario where a user deletes multiple areas in rapid succession. That's probably out of the scope of this PR though.